google/cloud/spanner_dbapi/cursor.py

```diff
@@ -290,6 +309,10 @@ def __iter__(self):
         return self._itr
 
     def fetchone(self):
+        """Fetch the next resulting row of the last ran query.
```

Comment:
Better use the [PEP-249](https://www.python.org/dev/peps/pep-0249/#fetchone) definition.